scss/texts.scss

```diff
@@ -0,0 +1,11 @@
+$gray : #828282;
+
+.text-1,
+.text-1-description {
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.text-1-description {
+  color: $gray;
+}
```

Comment:
Tip: You can configure vscode to add a new empty line. :)

Reply:
Done